packages/teams-js/src/public/monetization.ts

```diff
@@ -9,27 +10,41 @@ import { runtime } from './runtime';
  */
 export namespace monetization {
   /**
-   * @hidden
+   * @privateRemarks
    * Hide from docs
    * Data structure to represent a subscription plan.
    *
    * @internal
    */
   export interface PlanInfo {
     /**
-     * @hidden
+     * @privateRemarks
      * plan id
      */
     planId: string;
     /**
-     * @hidden
+     * @privateRemarks
      * term of the plan
      */
     term: string;
   }
 
   /**
-   * @hidden
+   * @privateRemarks
+   * Hide from docs
+   * Open dialog to start user's purchase experience
+   *
+   * @param planInfo optional parameter. It contains info of the subscription plan pushed to users.
+   * error can either contain an error of type SdkError, incase of an error, or null when get is successful
+   * @returns Promise that will be resolved when the operation has completed or rejected with SdkError value
+   *
+   * @internal
+   */
+  export function openPurchaseExperience(planInfo?: PlanInfo): Promise<void>;
+  /**
+   * @deprecated with TeamsJS v2 upgrades
```

Comment:
Please take a look at one of the other instances for how we need the @deprecated warnings to be formatted so it shows up in the API reference docs correctly (an example is localization.ts).

Reply:
Done